utils/vara-ui/src/components/alert/alert.tsx

```diff
@@ -1,13 +1,14 @@
 import { CSSProperties, ReactNode } from 'react';
 import cx from 'clsx';
-import { ReactComponent as CrossSVG } from '../../assets/images/cross.svg';
+import CrossSVG from '../../assets/images/cross.svg?react';
 import { Button } from '../button';
 import styles from './alert.module.scss';
 
 type Options = {
-  type: 'info' | 'error' | 'loading' | 'success';
+  type: 'info' | 'error' | 'loading' | 'success' | 'notification-warning' | 'notification-info';
```

Comment:
```suggestion
  type: 'info' | 'error' | 'loading' | 'success';
  variant?: 'alert' | 'notification';
```

Makes more sense to make it as a standalone variant?